src/svg-renderer.js

```diff
@@ -307,9 +307,11 @@ class SvgRenderer {
         // Enlarge the bbox from the largest found stroke width
         // This may have false-positives, but at least the bbox will always
         // contain the full graphic including strokes.
+        // If the width or height is zero however, don't enlarge since
+        // they won't have a stroke width that needs to be enlarged.
         const halfStrokeWidth = this._findLargestStrokeWidth(this._svgTag) / 2;
```

Comment:
Sorry! I misunderstood the intent of this PR. What if you set halfStrokeWidth to 0 if width or height are 0, so stroke width doesn't affect any aspect of the svg bounding box?

Reply:
Oh good point! That's simpler. One thing I'm wondering is if we'd want halfStrokeWidth to be larger than zero if the SVG had either a width or a height, but I guess if one of them is zero the SVG will necessarily not have much of a visible presence and not need to be scaled?

Reply:
Yeah, if the svg is changed at all in the paint editor the bounding box will be recalculated, so it doesn't matter too much.